packages/maestro/src/plugin/twig.js

```diff
@@ -0,0 +1,276 @@
+/**
+ * This is a transferred version of `larowlan/vite-plugin-twig-drupal
+ *
+ * TODO: reimplement with
```

Comment:
Transfer these improvements to a low priority issue so that we can eventually get to it. Add as much context as you can since it is fresh in your mind right now and it won't be in the future.

Reply:
Yup, @GGKapanadze could you do this?